components/x-audio/readme.md

```diff
@@ -23,16 +35,22 @@ The [`x-engine`][engine] module is used to inject your chosen runtime into the c
 
 ## Usage
 
-The components provided by this module are all functions that expect a map of [properties](#properties). They can be used with vanilla JavaScript or JSX (If you are not familiar check out [WTF is JSX][jsx-wtf] first). For example if you were writing your application using React you could use the component like this:
+This component exports both the plain UI component and the default is the redux-connected component, ready to play audio.
```

Comment:
Yeah same, _default is a factory function which returns the working audio player (under the hood this is the plain UI component hooked up to the redux store)_

Reply:
something like that maybe ?^ 🤔

Reply:
Yep, nice